app/models.py

```diff
@@ -0,0 +1,81 @@
+from typing import Optional
+from uuid import UUID, uuid4
+from datetime import datetime, timedelta, timezone
+from jose import JWTError, jwt
+from pydantic import BaseModel, EmailStr, Field
+from passlib.context import CryptContext
+
+#Set up password hashing context using bcrypt
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+#JWT settings
```

Comment:
Those things should be stored in .env file

Reply:
done.